### DISCUSSION TITLE
Can't remap `collapse_selection`

### DISCUSSION THREAD

**Discussion by pantos9000** [helix-editor/helix#10491] [General] [answered]
### Summary

I tried the following in my `config.toml`:
```
[keys.normal]
";" = "no_op"
"#" = "collapse_selection"
```

But it does not seem to change the behaviour, `;` still does the collapse, and `#` won't do anything. It seems to me the key is hard coded, but the docs gave me the impression that a remap should work.

Apologies if I misunderstood something, feel free to close if you think this is not a bug.

### Reproduction Steps

* create some selection with `v`
* press `#` -> nothing happens
* press `;` -> selection will collapse

### Helix log

log file is empty after doing reproduction steps

### Platform

Arch Linux

### Terminal Emulator

alacritty 0.13.2 (bb8ea18e)

### Installation Method

AUR (helix-git)

### Helix Version

helix 24.3 (1cce693b)

**Comment by pascalkuthe** [comment]
Can't reproduce, probably something with your setup where the cinfig file is nit being picked up

**Comment by pantos9000** [comment, marked as answer]
Nevermind, I'm stupid. Of course this has to go to `keys.select`, not `keys.normal`.

Sorry for the ruckus...
